e2e/cypress/integration/ui/settings/project_mgmt.spec.ts

```diff
@@ -75,171 +77,190 @@ describe('project management', () => {
-
-    cy.url().should('include', `/settings/projects/${projectID}/rules`);
-    cy.get('app-project-rules chef-page').should('be.visible');
-
-    cy.get('#create-name input').focus().type(ruleName, { delay: typeDelay })
-      .should('have.value', ruleName);
-    cy.get('[data-cy=edit-id]').click();
-
-    cy.get('#create-id input').should('have.value', ruleID);
-
-    // Verify correct attributes/operators are selectable for each resource
-    // Event
-    cy.get('#create-type-dropdown').select('Event');
-    cy.get('[data-cy=attribute-dropdown]').select('Chef Organization');
-    cy.get('[data-cy=attribute-dropdown]').select('Chef Server');
-    cy.get('[data-cy=operator-dropdown]').select('equals');
-    cy.get('[data-cy=operator-dropdown]').select('member of');
-
-    // Node
-    cy.get('#create-type-dropdown').select('Node');
-    const nodeAttributes = ['Chef Organization', 'Chef Server', 'Environment',
-      'Chef Role', 'Chef Tag', 'Chef Policy Name', 'Chef Policy Group'];
+    cy.get('[data-cy=create-name]').focus()
+      .type(projectName).should('have.value', projectName);
 
-    nodeAttributes.forEach((att: string) => {
-      cy.get('[data-cy=attribute-dropdown]').select(att);
-    });
+    cy.get('[data-cy=create-id]').should('not.be.visible');
+    cy.get('[data-cy=edit-button]').contains('Edit ID').click();
+    cy.get('[data-cy=id-label]').should('not.be.visible');
 
-    cy.get('[data-cy=operator-dropdown]').select('member of');
-    cy.get('[data-cy=operator-dropdown]').select('equals');
-    cy.get('[data-cy=rule-value] input').type('Chef Policy Group Foo');
+    // don't create associated project policies
+    cy.get('app-create-object-modal chef-checkbox').click();
 
-    cy.get('app-project-rules #right-buttons button').contains('Save Rule').click();
-    cy.get('app-project-rules chef-page').should('not.be.visible');
+    cy.get('[data-cy=create-id]').should('be.visible').clear()
+      .type(customWithoutPolsProjectID).should('have.value', customWithoutPolsProjectID);
 
-    cy.url().should('include', `/settings/projects/${projectID}`);
-    cy.get('app-project-details chef-td').contains(ruleID);
-    cy.get('app-project-details').contains('Edits are pending');
-  });
+    cy.get('[data-cy=save-button]').click();
+    cy.get('app-project-list chef-modal').should('not.be.visible');
 
-  itFlaky('displays a list of rules for a project', () => {
-    ['Name', 'ID', 'Resource Type', 'Conditions', 'Edits'].forEach((header) => {
-      cy.get('app-project-details chef-th').contains(header);
+    // verify success notification and then dismiss it
+    cy.get('app-notification.info').contains(`Created project ${customWithoutPolsProjectID}`);
+    cy.get('app-notification.info chef-icon').click();
+
+    cy.contains(customWithoutPolsProjectID).should('exist');
+
+    // verify no associated policies were generated
+    associatedPolicies.forEach((policy) => {
```

Comment:
This confused me for a moment; would be a bit more clear with this:
```suggestion
    associatedPolicySuffixes.forEach(suffix => {
```